pyrometheus/__init__.py

```diff
@@ -409,7 +435,7 @@ def get_concentrations(self, rho, Y):
         conctest = self.iwts * rho * Y
         zero = 0 * conctest[0]
         for i, conc in enumerate(conctest):
```

Comment:
Where is conc used in this loop?

Reply:
We *may* use it instead of `concs[index]`.  More testing is needed to make sure this flies.   I had trouble with it, and switched back to using indexed data structure in debugging.  I suggest we ignore this for now.